app/scripts/services/dimBungieService.factory.js

```diff
@@ -631,7 +631,12 @@
 
     /************************************************************************************************************************************/
 
-    function setLockState(item, store, lockState) {
+    function setItemState(item, store, lockState, type) {
+      switch (type) {
+      case 'lock': type = 'SetLockState'; break;
```

Comment:
fair enough, typically always tab in by default inside of the curlies, but can see why it might not be necessary here. TIL